src/frontend/src/styles/main.css

```diff
@@ -1731,7 +1731,6 @@ by all browsers (FF is missing) */
 
 .c-button:not([disabled]):focus,
```

Comment:
Instead I would suggest to add `:not([disabled])` to lines 1727-1730 and move them below lines 1732-1740. Since the text only button is an override of a normal button.

Removing the color as the current PR suggests might cause the browser stylesheet to color them differently e.g. when an anchor is styled as the button and has been visited.